server/src-lib/Hasura/GraphQL/Parser/Class.hs

```diff
@@ -122,6 +123,23 @@ askTableInfo tableName = do
   -- supposed to ensure that all dependencies are resolved.
   tableInfo `onNothing` throw500 ("askTableInfo: no info for " <>> tableName)
 
+-- | Helper function to get the table display name. A table may have an
+-- identifier configured with it. When the identifier exists, the GraphQL nodes
+-- that are generated according to the identifier. For example: Let's say,
+-- we have a table called `users address`, the name of the table is not GraphQL
+-- compliant so we configure the table with a GraphQL compliant name,
+-- say `users_address`
+-- The generated top-level nodes of this table will be like `users_address`,
+-- `insert_users_address` etc
+getTableDisplayName
```

Comment:
“Display name” is a bit vague for this concept, I think—“display” usually means “user-facing,” which this is not really in the context of the console. How about `getTableGraphQLName`, or `getTableGQLName` if the former is too verbose?

Reply:
I remember having lot of confusion while choosing the name for this function 😅 . `getTableGQLName` is much better. Thank you!